pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -168,6 +169,9 @@ def restore_training_state(self, checkpoint: Dict[str, Any]) -> None:
 
         self.restore_optimizers_and_schedulers()
 
+        # restore logging values
+        self.restore_result_collections()
```

Comment:
Since the loops hold the results, I'd call `restore_loops` and that should call `restore_results`

This would also be the structure in the checkpoint

cc: loops people @justusschock @awaelchli

Reply:
Done !